scripts/core/program.yml

```diff
@@ -264,6 +310,10 @@ desc: "Set a Program object specialization constant to a specific value"
 class: $xProgram
 name: SetSpecializationConstant
 ordinal: "0"
+analogue:
+    - "**clSetProgramSpecializationConstant**"
+details:
+    - "`hProgram` must not be an executable program, or a program created from a device binary."
 params:
```

Comment:
#269 is looking to add a kernel argument to enable better support for specialization constants on CUDA.